scripts/.air_s3.toml

```diff
@@ -0,0 +1,23 @@
+root = "."
```

Comment:
Instead of adding air toml file for every setup we would have, let's use https://github.com/cosmtrek/air#runtime-arguments for local development.